scripts/verifyCommitAndClose.ts

```diff
@@ -0,0 +1,92 @@
+import { Command } from "commander";
+import { BigNumber, ethers } from "ethers";
+import {
+  getDefaultProvider,
+  Networks,
+} from "./helpers/getDefaultEthersProvider";
+import colors from "colors";
+import hre from "hardhat";
+import moment from "moment";
+import deployments from "../constants/deployments.json";
+
+const program = new Command();
+program.version("0.0.1");
+program.requiredOption("-n, --network <network>", "Network", "mainnet");
+
+program.parse(process.argv);
+
+async function main() {
+  const network = program.network === "mainnet" ? "mainnet" : "kovan";
+
+  await verifyRollToNextOption(
+    deployments[network].RibbonETHCoveredCall,
```

Comment:
add "ETH",
add "WBTC",
add "ETH",

for first param of verifyRollToNextOption